File: Numbers/ReverseNumber.java
import java.util.Scanner;
public class ReverseNumber {
    public static int checkReverse(int n){
        int revNum=0;
        while(n!=0){
            int ld=n%10;
            revNum=(revNum*10)+ld;
            n=n/10;
        }
        return revNum;
   }
   public static void main(String[] args) {
    Scanner sc=new Scanner(System.in);
    System.out.print("Enter a number");
    int n=sc.nextInt();
    System.out.println("The reverse Number is:"+checkReverse(n));    
   }
}

File: Problem Solving Python/Numbers/Reverse_Number.py
def ReverseNumber(n):
    is_negative = n < 0
    n = abs(n)  # Work with the positive version of n
    revNum = 0
    while(n>0):
        ld=n%10
        revNum=(revNum*10)+ld
        n=n//10
    if(is_negative):
        return -revNum
    else:
        return revNum

if __name__=='__main__':
    n=int(input("Enter a number: "))
    print(f"The reverse number of {n} is:",ReverseNumber(n))
